docs/USING_PRO.md

```diff
@@ -82,6 +82,16 @@ slugger.slug('foo-1') // foo-1-2
 ...
 ```
 
+`slugger.slug` can also be called with the `dryrun` option for stateless operation:
+```js
+slugger.slug('foo')                    // foo
+slugger.slug('foo', { dryrun: true })  // foo-1
+slugger.slug('foo')                    // foo-1
+slugger.slug('foo', { dryrun: true })  // foo-2
+slugger.slug('foo')                    // foo-2
```

Comment:
Perhaps change this to the following:

```suggestion
slugger.slug('foo')                    // foo
slugger.slug('foo')                    // foo-1
slugger.slug('foo')                    // foo-2
slugger.slug('foo', { dryrun: true })  // foo-2
slugger.slug('foo', { dryrun: true })  // foo-2
slugger.slug('foo')                    // foo-3
```

Reply:
And maybe we should add that as a test ☝️

Reply:
Sure thing, I'll get this changed in the morning 👍

Reply:
Shouldn't the two dryruns return `"foo-3" ` since it should show what will happen on the next run?

Reply:
Yep, you're right, I updated my example.

More reason to update this since its not immediately clear if its supposed to return the last slug or next slug.

Maybe we can think of something better than `dryrun`? Can't think of anything off the top of my head that would describe the expected behavior 🤔

Reply:
I did previously think about calling it `stateless`?

Reply:
Let's keep `dryrun` 👍